tests/mpi/periodicity_08.mpirun=1.with_p4est=true.output

```diff
@@ -0,0 +1 @@
+
```

Comment:
Shouldn't there some output?

Reply:
now there is.

Reply:
Oh, I thought you wanted `pcout` to print to `output`.
I would prefer to see the output for `constraints.is_consistent_in_parallel()`